docs/general/builtins/scalar.rst

```diff
@@ -1393,6 +1393,37 @@ See below for an example::
     +-------------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-trunc:
+
+``trunc(number[, precision])``
+------------------------------
+
+`number` is the ``numeric`` value to be truncated.
+`precision` is an ``integer`` indicating the number of decimal places to
+preserve (defaults to 0).
```

Comment:
have a look at the other sections in this document. the style we've used for other functions is to have an intro para that explains what the function does and uses the argument literals in the explanation text

for example:

```
``power(a: number, b: number)``
-------------------------------

Returns the given argument ``a`` raised to the power of argument ``b``.
```

could you reword this so it follows the same style?